featurebyte/service/feature.py

```diff
@@ -114,7 +114,12 @@ async def prepare_feature_model(
                 "user_id": self.user.id,
                 "catalog_id": self.catalog_id,
             }
-        )
+        }
+        if sanitize_for_definition:
+            # add aggregation attributes to avoid triggering unnecessary aggregation attribute derivation
+            feature_dict["aggregation_ids"] = ["dummy_aggregation_id"]
+            feature_dict["aggregation_result_names"] = ["dummy_aggregation_result_name"]
+        return FeatureModel(**feature_dict)
```

Comment:
By skipping this, saving 300 features reduces from 7 mins to 5 mins (from 1.4s to 1.0s per feature).